tests/pipeline/base.py

```diff
@@ -95,24 +92,27 @@ def run_graph(self, graph, initial_workspace, mask=None):
         results : dict
             Mapping from termname -> computed result.
         """
+        def get_loader(c):
```

Comment:
This differs because we now raise if we even call `get_loader` with a column. Previously we allowed calling `get_loader`, but would then return an ExplodingObject.